shell/functions.sh

```diff
@@ -16,46 +16,70 @@ generate_uuid() {
     echo "$uuid"
 }
 
+check_env_vars() {
+    local required_vars=(
+        "GIT_REPO" "GIT_BRANCH" "GIT_HASH" "GIT_TAG"
+        "SPRYKER_TEST_ENVIRONMENT"
+    )
```

Comment:
shouldn't be all the variables from docs/Executing-Tests.md be there?

Reply:
Not all of them are strictly required.